contract/src/main/kotlin/com/ritense/valtimo/contract/annotation/AnnotatedClassResolver.kt

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.valtimo.contract.annotation
+
+import io.github.classgraph.ClassGraph
+import io.github.classgraph.ClassInfo
+import mu.KotlinLogging
+import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.context.ApplicationContext
+import java.lang.reflect.Method
+
+abstract class AnnotatedClassResolver(
```

Comment:
Cna you add javadocs so people know what this class is for?

Reply:
I would also like to see some tests for the methods in here.

Reply:
added